airflow/cli/commands/fastapi_api_command.py

```diff
@@ -101,6 +102,8 @@ def fastapi_api(args):
         )
 
         pid_file, _, _, _ = setup_locations("fastapi-api", pid=args.pid)
+
+        # TODO: Need to pass "apps" to create_app somehow
```

Comment:
I saw something like this to pass args to the function called, that might help, I haven't tested it:

```
gunicorn 'mypackage:build_app(foo="bar")'
```

Here https://stackoverflow.com/questions/8495367/using-additional-command-line-arguments-with-gunicorn

Reply:
Yup, done: https://github.com/apache/airflow/pull/43015/commits/8ef7b68da4eae34c097edeadb50809b09ce0fea3